.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -168,7 +177,17 @@ jobs:
           OS_APPLICATION_CREDENTIAL_SECRET: ${{ secrets.OS_APPLICATION_CREDENTIAL_SECRET }}
 
       - name: Terraform Apply
-        run: terraform apply -auto-approve
+        run: |
```

Comment:
Since this is just a command, we could try this retry action:
https://github.com/nick-fields/retry

Reply:
Just to clarify, I'm happy to merge as is. I don't think there's a huge benefit to using the custom action, but worth considering as an option.